iati/core/tests/test_resources.py

```diff
@@ -1,6 +1,7 @@
 """A module containing tests for the library implementation of accessing resources."""
 from lxml import etree
 import pytest
+from six import string_types
```

Comment:
`six` has caused problems (tests starting to fail or something otherwise not working) in both the Dashboard and Datastore. This may be to do with the mainly-2.7 codebases and out-of-date versions of dependencies, though I'm not sure.